docs/topics/mta-web-config-jira-connection.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-jira-connection_{context}"]
+= Configuring Jira connection
+
+You need to establish and authorize a connection to Jira for creating Jira issues from within the {ProductName} ({ProductShortName}) {WebName} and tracking application migrations.
+
+.Procedure
+
+. In *Administration* view, under *Issue Management*, click *Jira*.
++
+The *Jira configuration* page opens.
+. Click *Create new*.
++
+The *New instance* window opens.
+. Enter the following information:
+* Name of the instance
+* URL of the web interface of the account (??)
```

Comment:
URL of Jira account, the one which user will get from the page just after he will login to his Jira account, either Cloud or Datacenter/Server